### DISCUSSION TITLE
tip is too simple

### DISCUSSION THREAD

**Discussion by ct-groups** [dbeaver/dbeaver#15601] [General]
![image](https://user-images.githubusercontent.com/53501828/155270724-47d4c7b7-c0ab-4398-a42c-50cc17fc7eaa.png)

**Comment by LonwoLonwo** [comment]
Hello. In the case of MariaDB/MySQL databases, you have a special setting in the Preferences: "Show server help topics"

![изображение](https://github.com/user-attachments/assets/bd6fcd9d-6987-4174-b2ea-ff9c9191ed54)

![изображение](https://github.com/user-attachments/assets/72977185-1b56-4ba8-b1f4-8e5e4e58ebff)
